test/runtests.jl

```diff
@@ -17,4 +22,16 @@ using Test
     @test generate_mesh(control_file, verbose=true) isa String
   end
 
+  @testset "generate_mesh() with ABAQUS output" begin
+    control_file = joinpath(HOHQMesh.examples_dir(), "ABAQUS_IceCreamCone.control")
+    generate_mesh(control_file)
+    parse_mesh = abaqus_read_mesh(joinpath(outdir, "ABAQUS_IceCreamCone.inp"))
+    # set some reference values for comparison. These are the corner IDs for element 114
+    ref_IDs = [140, 141, 154, 153]
```

Comment:
```suggestion
    reference_ids = [140, 141, 154, 153]
```
More descriptive + snake case 😬